src/injected/styles/injected.scss

```diff
@@ -223,7 +207,6 @@
                 }
 
                 display: inline;
-                float: right;
```

Comment:
Was this style detected through a different process (not finding all references of a certain class)?

Reply:
This was found by VSCode telling me this property would never be applied due to the display property above.